misk-aws/src/main/kotlin/misk/dynamodb/DynamoDbHelpers.kt

```diff
@@ -0,0 +1,42 @@
+package misk.dynamodb
+
+import com.amazonaws.services.dynamodbv2.model.AmazonDynamoDBException
+import java.time.Duration
+import misk.backoff.Backoff
+import misk.backoff.DontRetryException
+import misk.backoff.ExponentialBackoff
+import misk.backoff.retry
+
+object DynamoDbHelpers {
+  fun getDefaultBackoff(): ExponentialBackoff {
+    return ExponentialBackoff(
+        baseDelay = Duration.ofMillis(10),
+        maxDelay = Duration.ofMillis(500),
+        jitter = Duration.ofMillis(5)
```

Comment:
I would put a larger value here like 200ms. See https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/